lib/active_admin/inputs/filters/date_range_input.rb

```diff
@@ -29,11 +29,11 @@ def input_html_options(input_name = gt_input_name, placeholder = gt_input_placeh
           rescue
             nil
           end
-          { size: 12,
-            class: "datepicker",
-            maxlength: 10,
-            placeholder: placeholder,
-            value: current_value ? current_value.strftime("%Y-%m-%d") : "" }
+          (options[:input_html] || {}).merge size: 12,
+                                             class: "datepicker",
+                                             maxlength: 10,
+                                             placeholder: placeholder,
+                                             value: current_value ? current_value.strftime("%Y-%m-%d") : ""
```

Comment:
By doing this, the defaults will override whatever it's specified as `options[:input_html]`, right? I don't think that's expected?

I think what we want is

```diff
diff --git a/lib/active_admin/inputs/filters/date_range_input.rb b/lib/active_admin/inputs/filters/date_range_input.rb
index 85a4c840c..399121ea0 100644
--- a/lib/active_admin/inputs/filters/date_range_input.rb
+++ b/lib/active_admin/inputs/filters/date_range_input.rb
@@ -33,7 +33,7 @@ module ActiveAdmin
             class: "datepicker",
             maxlength: 10,
             placeholder: placeholder,
-            value: current_value ? current_value.strftime("%Y-%m-%d") : "" }
+            value: current_value ? current_value.strftime("%Y-%m-%d") : "" }.merge(options[:input_html] || {})
         end
 
         def gt_input_placeholder
```

Reply:
If the previous review comment is correct, can you also add a test to make sure that whatever is specified as `input_html` overrides the defaults?